hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1415,6 +1415,19 @@ public void testBulkInsertRowWriterContinuousModeWithInlineClustering() throws E
         getTableServicesConfigs(2000, "false", "true", "3", "false", ""));
   }
 
+  @Test
+  public void testBulkInsertRowWriterContinuousModeWithInlineClusteringAmbiguousDates() throws Exception {
+    sparkSession.sqlContext().setConf("spark.sql.parquet.datetimeRebaseModeInWrite", "LEGACY");
+    sparkSession.sqlContext().setConf("spark.sql.avro.datetimeRebaseModeInWrite", "LEGACY");
+    sparkSession.sqlContext().setConf("spark.sql.parquet.int96RebaseModeInWrite", "LEGACY");
+    sparkSession.sqlContext().setConf("spark.sql.parquet.datetimeRebaseModeInRead", "LEGACY");
```

Comment:
This might be the use case we wanna to fix, is there any we can change this timestamp generation as configurable for `HoodieTestDataGenerator`, the global static variable has risk for configuration leak and it may be hard to maintain/debug in the future.

Reply:
yeah, good point. changed it to being in the constructor